### PR TITLE
LPS-112849 checking if there is a container to rendering the form

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
ci:test:forms

**Comment by linolaoj** [comment]
Hi @natocesarrego , the test failure was related. I'm going to push a commit and run the tests again.

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by linolaoj** [comment]
ci:test:forms

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.soy [hunk 1]
```diff
@@ -16,70 +17,72 @@
...
-				{param strings: $strings /}
-				{param submitLabel: $submitLabel ?: 'submit' /}
-				{param total: $total /}
-			{/call}
-		{else}
-			{foreach $page in $pages}
 				{call PageRenderer.render}
 					{param activePage: $activePage /}
 					{param editable: $editable /}
-					{param editingLanguageId: $editingLanguageId /}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.soy:62]: @linolaoj, why did you need to remove the editing language just for this case?

**Comment by natocesarrego** [comment]
@alinedoleron, can you please check this pull? I believe you can contribute, once this is a exclusive front-end pull.

Thanks in advance.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js [hunk 2]
```diff
@@ -60,6 +60,14 @@ class FormRenderer extends Component {
 	_handleFieldFocused(event) {
 		this.emit('fieldFocused', event);
 	}
+
+	_getDisplayableFn() {
+		return (
+			this.initialConfig_.readOnly ||
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js:66]: @linolaoj, from where this `initialConfig_` comes from?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.soy (hunk 1 above)
Comment (linolaoj) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.soy:62]: @natocesarrego , this was actually an accident. Sorry I missed that. I'll add it back.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js (hunk 2 above)
Comment (linolaoj) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js:66]: @natocesarrego , initialConfig_ holds the values passed when creating the FormRenderer. But I see there is a more clean way to access these values through the "state". I'll change that.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js [hunk 3]
```diff
@@ -60,6 +60,14 @@ class FormRenderer extends Component {
 	_handleFieldFocused(event) {
 		this.emit('fieldFocused', event);
 	}
+
+	_getDisplayableFn() {
```
[the remark this answers is not in the document]
Comment (linolaoj) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js:64]: @natocesarrego , the purpose of this method is to decide if the form should be displayed. I used the "Fn" because it seems to be the pattern when using a function to get an attribute value (git grep "Fn(" "*.js").

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by linolaoj** [comment]
ci:test:forms

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/alinedoleron/liferay-portal/pull/103. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
